ocs_ci/ocs/constants.py

```diff
@@ -619,6 +620,42 @@
 CEPH_CONFIG_DEBUG_LOG_LEVEL_CONFIGMAP = os.path.join(
     TEMPLATE_DEPLOYMENT_DIR, "ceph-debug-log-level-configmap.yaml"
 )
+# Multicluster related yamls
+ODF_MULTICLUSTER_ORCHESTRATOR = os.path.join(
+    TEMPLATE_MULTICLUSTER_DIR, "odf_multicluster_orchestrator.yaml"
+)
+MIRROR_PEER = os.path.join(TEMPLATE_MULTICLUSTER_DIR, "mirror_peer.yaml")
+VOLUME_REPLICATION_CLASS = os.path.join(
+    TEMPLATE_MULTICLUSTER_DIR, "volume_replication_class.yaml"
+)
+OPENSHIFT_DR_CLUSTER_OPERATOR = os.path.join(
+    TEMPLATE_MULTICLUSTER_DIR, "openshift_dr_cluster_operator.yaml"
+)
+OPENSHIFT_DR_HUB_OPERATOR = os.path.join(
+    TEMPLATE_MULTICLUSTER_DIR, "openshift_dr_hub_operator.yaml"
+)
+DR_POLICY_ACM_HUB = os.path.join(TEMPLATE_MULTICLUSTER_DIR, "dr_policy_acm_hub.yaml")
+ODR_S3_SECRET_YAML = os.path.join(TEMPLATE_MULTICLUSTER_DIR, "odr_s3_secret.yaml")
+
+# DR constants
+dr_default_namespace = "openshift-dr-systems"
```

Comment:
For constants we are following upper case for constants names but all your defined constants are lower case. Please follow the standard here.

Reply:
done, all constants are now uppercase